spinnaker_components/value_source/value_source.c

```diff
@@ -139,6 +145,9 @@ void c_main(void) {
       slots.current->length = pars.partial_block;
     }
 
+    // Compute the us delay between packets
+    us_delay = (pars.time_step - 100) / (pars.n_dims * 2);
```

Comment:
Is the ```* 2``` for safety? (i.e. to make sure we don't use more time than the time step)?  It looks to me like it means all the transmission will be spread out over the first half of the time step...